### ISSUE TITLE
Add flake8 support

### ISSUE THREAD

**Issue by robvdl** [jenkinsci/violations-plugin#51]
I would really like to add flake8 support, flake8 produces an output file that contains both pep8 and pyflakes violations, but in one file.

**Comment by tomasbjerre** [comment]
Do you have a sample report that we can use when creating a parser?

**Comment by anze3db** [comment]
@tomasbjerre flake8 can you the same format as pylint:

```
[smotko:~/myproject]$ flake8 --format=pylint myproject
myproject/__init__.py:7: [F401] 'db' imported but unused
myproject/__init__.py:7: [F401] 'logger' imported but unused
myproject/__main__.py:8: [E401] multiple imports on one line
myproject/__main__.py:15: [F401] 'ggrc' imported but unused
myproject/__main__.py:19: [E402] module level import not at top of file
myproject/__main__.py:24: [E265] block comment should start with '# '
myproject/__main__.py:25: [E265] block comment should start with '# '
myproject/__main__.py:26: [E265] block comment should start with '# '
myproject/app.py:6: [F401] 'sys' imported but unused
```

But the issue is that I can't seem to figure out how to use the violations plugin for both pylint and flake8 at the same time. Any ideas?

**Comment by asfaltboy** [comment]
@Smotko shouldn't flake8 already include pylint results? If not, I believe it's possible to list multiple report files in the violation field (comma separated):

![jenkins-4](https://cloud.githubusercontent.com/assets/321520/19644169/a097160a-99ee-11e6-8f23-fa130eb5ffe4.png)

Kindly let us know if using `--format=pylint` and the pylint violation field works for you.

**Comment by anze3db** [comment]
@asfaltboy No, flake8 does not include pylint. flake8 is a wrapper for PyFlakes, pep8 & Ned Batchelder’s McCabe script. 

My current workaround is to use the pep8 field which seems to be compatible with flake8's default output.

I didn't know you can specify multiple files, thanks for pointing that out!
